opal/Makefile.am

```diff
@@ -54,6 +56,7 @@ lib@OPAL_LIB_PREFIX@open_pal_la_LIBADD = \
         datatype/libdatatype.la \
         mca/base/libmca_base.la \
         util/libopalutil.la \
+        mca/common/verbs/libmca_common_verbs.la \
```

Comment:
This line should not be here, either.  Common libraries should not be linked into the core -- they are linked into individual components.